core/cli/commands/database.py

```
@@ -3196,3 +3196,283 @@ def link_code_smell(
+
+    try:
+        for col_name in collections:
+            col_type = _get_collection_type(col_name)
+            progress(f"[backfill-text] Scanning {col_name} ({col_type}) ...")
+
+            # Fetch all docs missing text
+            try:
+                docs: list[dict[str, Any]] = client.query(
+                    f"FOR d IN {col_name} FILTER d.text == null OR d.text == '' RETURN d"
+                )
+            except Exception as e:
```

Ruff (0.15.4)

[warning] Do not catch blind exception: `Exception`

(BLE001)